CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Main (unreleased)
   - `discovery.eureka` discovers targets from a Eureka Service Registry. (@spartan0x117)
   - `discovery.openstack` - service discovery for OpenStack. (@marctc)
   - `loki.write` now exposes basic WAL support. (@thepalbi)
+  - `loki.process` - added a new stage `non_indexed_labels` to attach non-indexed labels from extracted data to log line entry. (@vlad-diachenko)
```

Comment:
Can you make sure to update the reference documentation page for loki.process with the new stage?

Reply:
updated the docs